nullaway/src/main/java/com/uber/nullaway/handlers/BaseNoOpHandler.java

```diff
@@ -59,6 +63,20 @@ protected BaseNoOpHandler() {
     // We don't allow creating useless handlers, subclass to add real behavior.
   }
 
+  public static void reportMatch(
```

Comment:
I mentioned a utility class above.  Maybe this method can move there too?  And have default (package) visibility?